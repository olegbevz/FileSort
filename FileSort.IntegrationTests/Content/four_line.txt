52685. the best
335583. Cherry Apple is
223399. is the Cherry best Apple Apple is
423778. Banana Banana Cherry Something Banana Something
