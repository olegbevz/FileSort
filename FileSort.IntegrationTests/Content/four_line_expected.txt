423778. Banana Banana Cherry Something Banana Something
335583. Cherry Apple is
223399. is the Cherry best Apple Apple is
52685. the best
